curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657e928716b77b2277980276.md

```diff
@@ -9,7 +9,15 @@ dashedName: step-1
 
 In this project, you are going to learn about list comprehensions in Python by building a program that can take a `camelCase` or `PascalCase` formatted string and convert that to a `snake_case` formatted string. 
 
-List comprehensions in Python are a concise way to construct a list without using loops or the `.append()` method. Apart from being briefer, list comprehensions often run faster.
+In Python, a <dfn>List comprehension</dfn> is a construct that allows you to generate a new list by applying an expression to each item in an existing iterable and optionally filtering items with a condition. Apart from being briefer, list comprehensions often run faster.
```

Comment:
The definition has to be in the same paragraph as the dfn element. That means this is still the definition.

Reply:
But isn't the definition in the same paragraph?

I have been using this definition as a guide:
https://github.com/freeCodeCamp/freeCodeCamp/blob/dda4bb9d6a6c49ed626ed3bfa8dd69dfff9bc541/curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61b093179e7fc020b43b1bb3.md?plain=1#L10C1-L14C64